exercises/composed/basic/docker-compose.yml

```diff
@@ -1,11 +1,19 @@
-app:
-  image: ciao
+version: '2'
 
-proxy:
-  image: woollyams/nginx-reverse-proxy:201603101638
-  ports:
-    - 5678:8080
-  links:
```

Comment:
`links` implied a startup order, which we've lost here; we probably need a `depends_on` to compensate (see https://docs.docker.com/compose/compose-file/#/depends-on)